src/components/button/submit-button.tsx

```diff
@@ -0,0 +1,27 @@
+'use client'
+
+import React from 'react'
+import { Button, createStyles } from '@mantine/core'
+
+const useStyles = createStyles((theme) => ({
+  button: {
+    color: theme.black,
+    backgroundColor: theme.colors.red[6],
+    '&:hover': {
+      color: theme.black,
+      backgroundColor: theme.colors.red[8],
+    },
+  },
+}))
```

Comment:
Move styles into `index.styles.ts` file

Reply:
There was no `index.styles.ts` file unless you meant for me to make one. If that was the case, I made a `submit-button.styles.ts` file to follow the naming convention we already had in place.